### ISSUE TITLE
Partial packages are being copied to Pictor

### ISSUE THREAD

**Issue by bonniegee** [RockefellerArchiveCenter/ursa_major#153]
Pictor (the second location that Ursa Major is delivering a package to, see #144) occasionally throws errors when unpacking and validating a package sent from Ursa Major. Upon further inspection, it's clear that the package in Pictor's `src` directory is much smaller than the equivalent package in Fornax's `src` directory (the first location, which is symlinked to Ursa Major's `dest` directory). For example:

<img width="689" alt="Screen Shot 2022-01-17 at 09 32 35" src="https://user-images.githubusercontent.com/8737364/149957479-ffc2b52e-add3-424e-b96c-55bcd5107bb2.png">

If the package from Fornax's `src` directory is copied to Pictor's, the errors are resolved.

I am not 100% sure why this is happening, but my hunch is that Ursa Major is re-starting the Deliver Package service over itself. We may want to start with the approach of an "in process" status that the service checks for.

**Comment by bonniegee** [comment]
An update on this (to explain the logic of an incoming PR): I forgot that Ursa Major was still processing a list of bags (we've changed the other microservices to mostly process one bag at a time). I checked a few of the bags that were causing this issue, and they were part of multi-bag deliveries. So I think we can start by delivering one bag at a time, and if needed we can introduce the in-process status.
